map_view/map-rotation/src/main/java/com/esri/samples/map_rotation/MapRotationSample.java

```diff
@@ -67,41 +65,25 @@ public void start(Stage stage) {
       String yourAPIKey = System.getProperty("apiKey");
       ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
 
-      // create a slider with a range of 360 units and set it to 5 degrees
-      slider = new Slider(-180.0, 180.0, 5.0);
-      styleSlider();
-
-      // listen for the value in the slider to change
-      slider.valueProperty().addListener(e -> {
-        // rotate map view based on new value in slider
-        mapView.setViewpointRotationAsync(slider.getValue());
-      });
-
       // create a map with the streets basemap style
-      map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
-
-      // enable slider when map view is done loading
-      map.addDoneLoadingListener(() -> {
-        if (map.getLoadStatus() == LoadStatus.LOADED) {
-          slider.setDisable(false);
-        } else {
-          Alert alert = new Alert(Alert.AlertType.ERROR, "Map Failed to Load!");
-          alert.show();
-        }
-      });
+      ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS);
 
       // create a map view and set the map to it
       mapView = new MapView();
-      // disable keyboard navigation to focus on slider/compass map rotation interaction
-      mapView.setEnableKeyboardNavigation(false);
       mapView.setMap(map);
 
-      // create a compass to show the current heading when rotated
-      Compass compass = new Compass(mapView);
+      // create a compass to show the direction of north
+      var compass = new Compass(mapView);
+
+      // create labels for instructions and map rotation
+      Label instructionsLabel = new Label("Press the A and D keys to rotate the map.");
+      instructionsLabel.setTextFill(Color.WHITE);
+      Label rotationLabel = new Label("Current rotation: ");
+      rotationLabel.setTextFill(Color.WHITE);
```

Comment:
```suggestion
      var instructionsLabel = new Label("Press the A and D keys to rotate the map.");
      instructionsLabel.setTextFill(Color.WHITE);
      var rotationLabel = new Label("Current rotation: ");
      rotationLabel.setTextFill(Color.WHITE);
```
Let's be consistent and use `var` across the sample where it makes sense, if we've already updated in one place e.g. compass